packages/contact-manager-rest-api/src/contact-manager-api-server.ts

```diff
@@ -0,0 +1,85 @@
+import { agentContext } from '@sphereon/ssi-sdk.core'
+import { TAgent } from '@veramo/core'
+
+import express, { Express, Router } from 'express'
+import { ContactManagerMRestApiFeatureEnum, IContactManagerAPIEndpointOpts, IRequiredPlugins } from './types'
+import {
+  identityReadEndpoints,
+  partyReadEndpoints,
+  partyTypeReadEndpoints,
+  partyWriteEndpoints
+} from './api-functions'
+import { copyGlobalAuthToEndpoints, ExpressSupport } from '@sphereon/ssi-express-support'
+
+export class ContactManagerApiServer {
+  private readonly _express: Express
+  private readonly _agent: TAgent<IRequiredPlugins>
+  private readonly _opts?: IContactManagerAPIEndpointOpts
+  private readonly _router: Router
+
+  constructor(args: { agent: TAgent<IRequiredPlugins>; expressSupport: ExpressSupport; opts?: IContactManagerAPIEndpointOpts }) {
+    const { agent, opts } = args
+    this._agent = agent
+    copyGlobalAuthToEndpoints({ opts, keys: ['contactRead', 'contactTypeRead', 'identityRead'] })
+    const enableFeatures = opts?.enableFeatures ?? [
+      ContactManagerMRestApiFeatureEnum.party_read,
```

Comment:
Do not start incorporating enums all of a sudden. The strings where typed to begin with, so you cannot just have a string that is not in the type.
Enums also do not play really nice with payloads and/or env vars. Please ask questions if you see me doing one thing and you start doing another thing. There is always are reason why I do something!

Reply:
removed the enum and used literal strings